pkg/scheduler/framework/runtime/framework_test.go

```diff
@@ -1591,19 +1580,24 @@ func TestFilterPlugins(t *testing.T) {
 					name: "TestPlugin1",
 					inj:  injectedResult{FilterStatus: int(framework.Success)},
 				},
-
 				{
 					name: "TestPlugin2",
 					inj:  injectedResult{FilterStatus: int(framework.Unschedulable)},
 				},
 			},
 			wantStatus: framework.NewStatus(framework.Unschedulable, injectFilterReason).WithFailedPlugin("TestPlugin2"),
-			wantStatusMap: framework.PluginToStatus{
-				"TestPlugin2": framework.NewStatus(framework.Unschedulable, injectFilterReason).WithFailedPlugin("TestPlugin2"),
-			},
 		},
 	}
 
+	cmpOpts := []cmp.Option{
```

Comment:
We can't use `cmp.Diff(wantStatus, gotStatus)` directly because `status.Equal()` will compare the error via `errors.Is()`. 
```
// An error is considered to match a target if it is equal to that target or if
// it implements a method Is(error) bool such that Is(target) returns true.
```

We met this problem in several test cases, but I haven't thought of any better ideas yet. Some experiments like:
1) Compare the error message for simplify, AFAIK `status.Equal()` only used in tests.
2) Introduce a new error type embedded to status which implements `func (m MyError) Is(target error) bool`.